datacapturing/src/main/java/de/cyface/datacapturing/backend/CapturingProcess.java

```diff
@@ -169,11 +178,13 @@ public abstract class CapturingProcess implements SensorEventListener, LocationL
         Sensor accelerometer = sensorService.getDefaultSensor(Sensor.TYPE_ACCELEROMETER);
         Sensor gyroscope = sensorService.getDefaultSensor(Sensor.TYPE_GYROSCOPE);
         Sensor magnetometer = sensorService.getDefaultSensor(Sensor.TYPE_MAGNETIC_FIELD);
+        Sensor barometer = sensorService.getDefaultSensor(Sensor.TYPE_PRESSURE);
         sensorEventHandlerThread.start();
         Handler sensorEventHandler = new Handler(sensorEventHandlerThread.getLooper());
-        registerSensor(accelerometer, sensorEventHandler);
-        registerSensor(gyroscope, sensorEventHandler);
-        registerSensor(magnetometer, sensorEventHandler);
+        registerSensor(accelerometer, sensorEventHandler, delayBetweenSensorEventsInMicroseconds);
+        registerSensor(gyroscope, sensorEventHandler, delayBetweenSensorEventsInMicroseconds);
+        registerSensor(magnetometer, sensorEventHandler, delayBetweenSensorEventsInMicroseconds);
+        registerSensor(barometer, sensorEventHandler, 1_000_000);
```

Comment:
If you do it like this, there is no need to store the delay in a property any longer, I guess.